src/containers/Menu/Menu.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./Menu.scss";
+import { Link } from "react-router-dom";
+import { AppRoutes } from "../../utils/consts";
+
+const Menu = () => {
+  return (
+    <div className="main-menu">
+      <ul>
+        <li>
+          <Link to={AppRoutes.upcomings} className="nav-item">
+            {AppRoutes.upcomings.substring(1)}
```

Comment:
просто рендерь строку, зачем тебе такие сложности?)
```suggestion
            Upcomings
```